easybuild/easyconfigs/g/GULP/gulp-6.0-mkgulp.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/gulp-6.0.org/Src/mkgulp b/gulp-6.0/Src/mkgulp
```

Comment:
Please add a description of what this patch does, and who the author is (you presumably). I think the patch could be named something with `fftw_blas_linking` or something instead of `mkgulp`.